include/exadg/time_integration/time_int_gen_alpha_base.h

```diff
@@ -53,17 +53,39 @@ class TimeIntGenAlphaBase : public TimeIntBase
   set_current_time_step_size(double const & time_step_size) final;
 
   double
-  get_scaling_factor_mass() const;
+  get_scaling_factor_mass_from_acceleration() const;
+
+  double
+  get_scaling_factor_mass_from_velocity() const;
 
 protected:
   double
   get_mid_time() const;
 
+  /*
+   * computes the finite element vector corresponding to the *remainder* of the acceleration term,
+   * int( phi_u, density * D^2/Dt^2(d) )_Omega, which does *not* depend on the displacement and is
+   * shifted to the right hand side. Applying the mass operator to this vector allows computing the
+   * contribution to the rhs once before up front before solving the linear or nonlinear system.
+   */
+  void
+  compute_const_vector_acceleration_remainder(VectorType &       const_vector,
+                                              VectorType const & displacement_n,
+                                              VectorType const & velocity_n,
+                                              VectorType const & acceleration_n) const;
+
+  /*
+   * computes the finite element vector corresponding to the *remainder* of the velocity term,
+   * int( phi_u, density * weak_damping_coefficient * D/Dt(d) )_Omega, which does *not* depend on
```

Comment:
I think "int" can be skipped, since this is indicated through the brackets. "phi_u" is somewhat unclear.